README.md

```diff
@@ -161,10 +142,13 @@ You will need to install the EB CLI tools using PIP.
 
 ```
 pyenv deactivate                 # to exit the virtual environment
-sudo pip install awsebcli        # install the eb cli tools
+pip install --user awsebcli        # install the eb cli tools
```

Comment:
Is the deactivate line needed as the virtualenv will only be used if the command is prefixed with `pipenv run` now?